config/config.go

```diff
@@ -0,0 +1,44 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+//  New creates a Config from a JSON file.
+func New(path string) (*Config, error) {
+	file, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("sal: opening client config file: %s", err)
```

Comment:
I think `sal: ` should be removed it results in a pretty weird looking error message IE

```bash
gosal did not complete: loadin config file test.json: sal: opening client config file: open test.json: no such file or directory
```

If this is normal in golang let me know but it's weird... It also doesn't match some other areas of the code but it could be a stylistic choice?

Reply:
it should be prefix with `config`. It's contextualizing where the error comes from. 
Example from stdlib would be `tls: unexpected EOF`